src/main/scala/io/confluent/examples/streams/algebird/CMSStoreChangeLogger.scala

```diff
@@ -44,7 +44,7 @@ class CMSStoreChangeLogger[K, V](val storeName: String,
     if (collector != null) {
       val keySerializer = serialization.keySerializer
       val valueSerializer = serialization.valueSerializer
-      collector.send(this.topic, key, value, this.partition, timestamp, keySerializer, valueSerializer)
+      collector.send(this.topic, key, value, null, this.partition, timestamp, keySerializer, valueSerializer)
```

Comment:
using `null` here as `context.headers()` results in an `IllegalStateException` as the `RecordContext` is null since the key and value is passed to the `RecordCollector` here on store flush.